main/test/ca/uwaterloo/flix/library/TestDelayMap.flix

```diff
@@ -112,4 +112,471 @@ namespace TestDelayMap {
         let _ = DelayMap.toList(m);
         listBeforeEvaluation == Nil and List.reverse(deref l) == "a" :: "a" :: "a" :: "b" :: "b" :: "b" :: Nil
 
+
+    /////////////////////////////////////////////////////////////////////////////
+    // map (pure)                                                              //
+    /////////////////////////////////////////////////////////////////////////////
+
+    @test
+    def mapPure01(): Bool =
+        Map#{}: Map[Unit, Int32] |> Map.toDelayMap |> DelayMap.map(v -> 3 * v) |> DelayMap.toMap == Map#{}
```

Comment:
I think this is too hard to read on 1 line. Can you break it down into multiple lines? Thanks!